.github/workflows/test-package.yml

```diff
@@ -76,42 +82,23 @@ jobs:
           parallel: true
           path-to-lcov: ./jni_gen/coverage/lcov.info
 
-  test_jni_gen_test_plugin:
-    needs: test_jni_gen
-    runs-on: ${{ matrix.os }}
+  ## TODO: More minimal test on windows after fixing dev dependency.
```

Comment:
The format we typically use is `TODO(#15):`. Makes it easy to see that every TODO has a github issue